projects/models.nix

```
@@ -96,6 +107,7 @@ rec {
         programs = optionalAttrs (option programType);
         services = optionalAttrs (option serviceType);
       };
+      blobs = option (list blobType);
```

```suggestion
      blobs = optionalAttrs blobType;
```

There's no inherent ordering, it should be an attrset. Also lists don't allow access by symbolic name